spark/dl/src/main/scala/com/intel/analytics/bigdl/utils/intermediate/BlasToIR.scala

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.utils.intermediate
+
+import com.intel.analytics.bigdl._
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, Activity}
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import scala.reflect.ClassTag
+
+private[bigdl] class BlasToIR[T: ClassTag] extends ConvertBase[Module[T], IRElement[T]]{
+
+  // reminder: some undefined IR operations can be presented by IRBlasModule
+  override def enableConvertLayer(layer: Module[T]): Boolean = {
+    val layerName = layer.getClass.getSimpleName
+    val className = "com.intel.analytics.bigdl.utils.intermediate.IR" + layerName
+    val cls = ReflectUtils.classFound(className)
+    if ( cls != null) return true
+    if (layer.isInstanceOf[AbstractModule[Activity, Activity, T]]) true
```

Comment:
you can just write `layer.isInstanceOf[AbstractModule[Activity, Activity, T]]`  no need to write if else

Reply:
was just hanging around, sorry to interrupt you guys, but can't resist to offer:

```
val layerName = layer.getClass.getSimpleName
val className = s"com.intel.analytics.bigdl.utils.intermediate.IR${layerName}"

ReflectUtils.classFound(className) != null || 
	layer.isInstanceOf[AbstractModule[Activity, Activity, T]]
```

Reply:
yeah, changed, thanks for your comments.